CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 1.44.0
```

Comment:
This should probably be 1.43.1

Reply:
So actually this is bumping from the pre-Fastlane version, so I think it merits a minor version bump. But I'm fine with either one.